config/localized-routes.php

```diff
@@ -15,22 +13,20 @@
         'es' => 'Español',
         'pl' => 'Polski',
         'pt_BR' => 'Português (Brasil)',
+        'de' => 'Deutsch',
     ],
-
```

Comment:
Why delete this line and the ones below it?

Reply:
You are right, this happened due to auto-formatting. I readded the lines.